Adds pencil cases to the loadout and gives explorer a light first-aid kit (in their lockers).

Flying-loulou opened the pull request:
:cl: Flying_loulou
rscadd: Adds pencil cases to the loadout (utility tab).
rscadd: Adds light first-aid kits to the exploration department's lockers.
/:cl:

Flying-loulou commented:
Following Spookerton's requests, nerfed the first aid kit a bit by taking out 1 roll of gauze and the splints.

MistakeNot4892 commented:
Loadout was intended for character flavour and a bit of cosmetic customisation. Giving everyone access to medkits and toolkits irrespective of their job loadout or any real need is just contributing to the total lack of scarcity on every Bay map since mid to late Exodus.

If there is no issue accessing these items on the map, then they don't need to be in loadout.

MistakeNot4892 commented on code/game/objects/items/weapons/storage/fancy.dm:
```diff
@@ -100,6 +100,28 @@
+ * Pencil Case
+ */
+
+/obj/item/weapon/storage/fancy/Pencilcase
```
Paths should generally be lowercase.

MistakeNot4892 commented on code/game/objects/items/weapons/storage/toolbox.dm:
```diff
@@ -65,3 +65,25 @@
+
+/obj/item/weapon/storage/toolpouch
+	name = "tool pouch"
+	desc = "It's a small pouch designed to carry a few tools around.."
```
Double period on the end.

Flying-loulou commented:
> Loadout was intended for character flavour and a bit of cosmetic customisation. Giving everyone access to medkits and toolkits irrespective of their job loadout or any real need is just contributing to the total lack of scarcity on every Bay map since mid to late Exodus.
> 
> If there is no issue accessing these items on the map, then they don't need to be in loadout.

The goal of this was to get people something for their side training (example : a master at arms, or an explorer picking basic medicine would potentially be given a first aid kit for him to keep, and use if necessary).

Flying-loulou committed:
Adds pencil cases to the loadout and gives explorer a light first-aid kit (in their lockers).

Flying-loulou commented:
Following requests, deleted the tool pouch, moved the light first-aid kit out of the loadout (it's only findable in explorer's lockers for now), and just left the pencil case untouched.

This is ready for review.

RenardQueenston commented:
The open pencil case has no sprite. Wtf?
